gen/definitions/profile_parcels/transport_cellular_profile.yaml

```diff
@@ -0,0 +1,62 @@
+---
+name: Transport Cellular Profile
+rest_endpoint: /v1/feature-profile/sdwan/transport/%v/cellular-profile
+minimum_version: 20.12.0
+test_tags: [SDWAN_2012]
+parcel_type: feature
+skip_minimum_test: true
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_transport_feature_profile.test.id
+  - model_name: configType
```

Comment:
This field seems to be only in API and not in the UI, also, it looks like in API is mandatory?

Reply:
In cases like this where field is not present in UI, mandatory in API, enum and has only one possible value, maybe we should set it by default it non-eSim (if user doesn't specify something else in the future)?

Reply:
This is already included by default, is not user definable and does not show in documentation